src/clarity-angular/data/datagrid/datagrid.ts

```diff
@@ -111,6 +111,7 @@ export class Datagrid implements AfterContentInit, AfterViewInit, OnDestroy {
                 state.filters.push(filter);
             }
         }
+        console.log(state);
```

Comment:
You missed a `console.log()` during cleanup.

Reply:
Thanks for catching it. Will remove.